spring-xd-dirt/src/main/java/org/springframework/xd/dirt/module/MD5StalenessCheck.java

```diff
@@ -0,0 +1,71 @@
+/*
+ *
+ *  * Copyright 2011-2014 the original author or authors.
```

Comment:
Copyright range should include the year the class was introduced and any subsequent years. Was this copied from code that originated in 2011, and if not, what's the rationale?

Also, the formatting looks off here with double asterisks per line.